conf/machine/include/rpi-base.inc

```diff
@@ -18,6 +18,7 @@ RPI_KERNEL_DEVICETREE_OVERLAYS ?= " \
     overlays/overlay_map.dtb \
     overlays/at86rf233.dtbo \
     overlays/disable-bt.dtbo \
+    overlays/disable-wifi.dtbo \
```

Comment:
does it change just the out of box defaults ?

Reply:
No, the image before and after the change is the same.  Except that the dtbo file is in the /boot partition.  But not used.

If you would like to have the Wifi disabled you should add into the kas file:
`RPI_EXTRA_CONFIG = "dtoverlay=disable-wifi"`

If you would like to have Bluetooth and Wifi disabled you can use
`RPI_EXTRA_CONFIG = "dtoverlay=disable-wifi\ndtoverlay=disable-bt"`

Reply:
yeah thanks for confirming.